generators/templates/standalone-README.md

```diff
@@ -1,17 +1,52 @@
-<%= readmeName %> Element
+# <%= readmeName %> Element
+A brief description of the web component, it's purpose and goals.
+
+## Usage
+Describe how best to use this web component along with best practices.
+
+```html
+<<%= elementName %>>
+    <h2>This is <%= elementName %></h2>
+    <!-- Slots --><% for(let i = 0; i < slots.length; i++) { %>
+    <div slot="<%= slots[i] %>"><%= slots[i] %> slot</div><% } %>
+</<%= elementName %>>
+```
+
+### Accessibility
+Explain how this component meets accessibility standards.
+
+## Slots
+
+- `namedSlot`: Describe each available slot and best practices around what markup it can contain.
+
+## Attributes
+
+- `attr`: Describe each available attribute and what function is serves.
+
+## Events
+Describe any events that are accessible external to the web component. There is no need to describe all the internal-only functions.
+
+## Dependencies
+Describe any dependent elements or libraries here too.
+
+Make sure you have [Polyserve][polyserve] and [Web Component Tester][web-component-tester] installed.
+
+    `npm install -g polyserve web-component-tester`
 
 ## Dev
 
-    npm start
+    `npm start`
```

Comment:
The indentation creates code blocks, so the backticks aren't necessary here.

Reply:
Addressed in https://github.com/patternfly/generator-pfelement/pull/32/commits/a391704428211bb53533a68d5d2d1f73901f43c9

Reply:
:+1: